avocado_vt/options.py

```diff
@@ -89,13 +89,14 @@ def __init__(self, options):
         set_opt(self.options, 'vt_smp',
                 settings.get_value('vt.qemu', 'smp', default='2'))
         set_opt(self.options, 'vt_image_type',
-                settings.get_value('vt.qemu', 'image_type', default='qcow2'))
+                settings.get_value('vt.qemu', 'image_type',
+                                   default=SUPPORTED_IMAGE_TYPES[0]))
         set_opt(self.options, 'vt_nic_model',
                 settings.get_value('vt.qemu', 'nic_model',
```

Comment:
This settings is the new module, right? The new module has no `get_value()`, method. Not sure if understood here.

Reply:
Marking as resorved as per https://github.com/avocado-framework/avocado-vt/pull/2708#discussion_r469894673